development/image-syncer/README.md

```diff
@@ -12,27 +12,40 @@ Syncing process steps:
 4. Check if image exists in target.
 5. If image does not exist, re-tag image and push to target registry.  
 If image exists: compare IDs. If they are different, synchronization is interrupted. It means that something is wrong and the source image was changed (this should not happen).
+6. If the image signing is enabled, verify if the image matches with the given KMS key. If the signature does not exist, sign the image and push the signature to the target repository. If the signature is not valid with the provided KMS key, synchronisation is interrupted.
 
-These steps guarantee that images in our registry are immutable.
+These steps guarantee that images in our registry are immutable and verified.
 
 ## Usage
 
 To run image-syncer, use:
 ```bash
 go run main.go \ 
     --images-file={PATH_TO_A_YAML_FILE_CONTAINING_SYNC_DEFINITION} \
-    --target-key-file={PATH_TO_A_JSON_KEY_FILE} \
-    --dry-run=true
+    --target-repo-auth-key={PATH_TO_A_JSON_KEY_FILE} \
+    --dry-run=true \
+    --kms-key=gcpkms://path/to/key
 ```
 
+### Image signing
+
+image-syncer supports safe image signing using [cosign](https://github.com/sigstore/cosign).
+However, it supports only the signing using KMS providers supported by sigstore package, although only GCP KMS was tested.
```

Comment:
```suggestion
It supports only the signing that uses KMS providers supported by the sigstore package. For more information, refer to the [KMS documentation](https://github.com/sigstore/cosign/blob/main/KMS.md) in the `cosign` repository.
```

Reply:
`although only GCP KMS was tested` -> I'd remove that and I'd link to the external documentation instead. This will make us immune to changes on their side 😉